osu.Game/Overlays/Mods/ModState.cs

```diff
@@ -24,6 +24,13 @@ public class ModState
         /// </summary>
         public BindableBool Active { get; } = new BindableBool();
 
+        /// <summary>
+        /// Whether the mod requires further customisation.
+        /// This flag is read by the <see cref="ModSelectOverlay"/> to determine if the customisation panel should be opened after a mod change
+        /// and cleared after reading.
```

Comment:
I'm not sure I immediately understand why it should be cleared. Is the intention that after configuring once it shouldn't be again? Maybe `PendingConfiguration` is a better variable name?

Reply:
It's cleared so that the configuration panel isn't opened again every time you select another mod, correct.